### PR TITLE
-M option

### PR DESCRIPTION
**Pull request by MichielPater**
-M: do not print "mounted on"

### CODE REVIEW

**Comment by Rolinh** [comment]
Hi Michiel,

I think that you mean
```c
static void text_disp_ln_end(void);
/* ... */
static void
text_disp_ln_end(void)
{
        (void)printf("\n");
}
```

Instead of
```c
static void text_disp_ln_end();
/* ... */
static void
text_disp_ln_end()
{
        (void)printf("\n");
}
```
And so on for every `_disp_ln_end()` export function.
(C != C++ :wink: )

LGTM otherwise.

File: src/export/html.c [hunk 1]
```diff
@@ -358,6 +361,7 @@ static void
 html_disp_mount(const char *dir)
 {
 	(void)printf("\t  <td>%s</td>", dir);
+	(void)puts("");
```
Comment (Rolinh) [inline src/export/html.c]: `(void)printf("\t  <td>%s</td>\n", dir);` perhaps?

File: src/export/html.c [hunk 2]
```diff
@@ -367,7 +371,8 @@ html_disp_mount(const char *dir)
 static void
 html_disp_mopt(const char *opts)
 {
-	(void)printf("\n\t  <td>%s</td>", opts);
+	(void)printf("\t  <td>%s</td>", opts);
+	(void)puts("");
```
Comment (Rolinh) [inline src/export/html.c]: `(void)printf("\t  <td>%s</td>\n", opts);` perhaps?

**Comment by MichielPater** [comment]
Updated, tested and works.

**Comment by Rolinh** [comment]
I have a suggestion though: I think it makes sense to take into account this new "-M" option for the auto-adjust feature (see `auto_adjust()` in `src/util.c`), probably around the `Tflag` check.

File: po/fr.po [hunk 3]
```diff
@@ -185,100 +188,101 @@ msgid "Could not assign value from configuration file: %s\n"
...
+#: src/util.c:516 src/export/html.c:169 src/export/tex.c:156
+#: src/export/text.c:128
 msgid "MOUNT OPTIONS"
 msgstr "OPTIONS DE MONTAGE"
 
-#: src/util.c:540
-msgid "Cannot compute required width"
+#: src/util.c:541
+#, fuzzy
+msgid "Cannot compute required width\n"
```
Comment (Rolinh) [inline po/fr.po]: Why adding a `\n` here?

**Comment by MichielPater** [comment]
Should be better now. There was supposed to be a `\n`

**Comment by Rolinh** [comment]
I think you mean to update `update_maxwidth()` for -M flag support instead of adding your checks with 0f5f77f44b61bd03887d6fa6219d354b22a01544.

**Comment by MichielPater** [comment]
Updated.

File: src/util.c [hunk 4]
```diff
@@ -637,9 +637,16 @@ auto_adjust(int tty_width)
...
+	if (!Mflag) {
+		Mflag = 1;
+		req_width -= max.mntdir;
+		if (tty_width >= req_width)
+			return;
+	}
 	if (iflag) {
 		iflag = 0;
 		req_width -= max.nbinodes;
+		req_width -= max.avinodes;
```
Comment (Rolinh) [inline src/util.c]: Nice catch!

**Comment by Rolinh** [comment]
Thanks!

**Pull request merged**